packages/lime-system/files/usr/lib/lua/lime/network.lua

```diff
@@ -260,7 +268,41 @@ function network.scandevices(specificIfaces)
 			--! pluggable ethernet dongles.
 			utils.log( "network.scandevices.dev_parser found plain Ethernet " ..
 			           "device %s", dev )
-		elseif dev:match("^wlan%d+"..wireless.wifiModeSeparator.."%w+$") then
+		end
+
+		if dev:match("^eth%d+%.%d+$") then
```

Comment:
This code block was removed in #1154. Maybe it should not be here?

Reply:
yeah this got pulled back in by rebase, this PR should not touch this stuff at all